workitem/link/link_repository.go

```diff
@@ -288,7 +288,7 @@ func (r *GormWorkItemLinkRepository) Save(ctx context.Context, lt app.WorkItemLi
 func (r *GormWorkItemLinkRepository) ListWorkItemChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
 	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
 	wilt := WorkItemLinkType{}
-	r.db.Where("forward_name = ?", "relates to").First(&wilt)
+	r.db.Where("forward_name = ?", "parent of").First(&wilt)
```

Comment:
I think we should get rid of this whole searching for a built-in link type and instead have the standard link types generated with fixed UUIDs. But that would be a mid-term goal and not relevant for this PR. @xcoulon what do you think?